lang/src/org/partiql/lang/eval/physical/window/WindowFunctionSignature.kt

```diff
@@ -0,0 +1,12 @@
+package org.partiql.lang.eval.physical.window
+
+/**
+ * Window function signature
+ *
+ * For now just holding function name
+ *
+ * In the future we can add additional parameter such as frame information here.
+ */
+class WindowFunctionSignature(
```

Comment:
Will there be an eventual use case for the signature? Would it be of use to make this a data class?

Reply:
Potentially we might be able to put the frame clause here in the future. If so, we can potentially group the window functions by partition, i.e., each window operator will have a list of window function that shares the same window partition but may differ at window frame. This is an optimization work so we don't need to recalculate the partition multiple times.